e2e/testsuite/testsuite.go

```diff
@@ -208,20 +229,17 @@ func (s *E2ETestSuite) SetupPath(clientOpts ibc.CreateClientOptions, channelOpts
 		err = test.WaitForBlocks(ctx, 1, chainA, chainB)
 		s.Require().NoError(err)
 
-		channelsA, err := r.GetChannels(ctx, s.GetRelayerExecReporter(), chainA.Config().ChainID)
-		s.Require().NoError(err)
+		s.testPaths[s.T().Name()] = append(s.testPaths[s.T().Name()], pathName)
 
-		channelsB, err := r.GetChannels(ctx, s.GetRelayerExecReporter(), chainB.Config().ChainID)
-		s.Require().NoError(err)
+		for _, c := range []ibc.Chain{chainA, chainB} {
+			channels, err := r.GetChannels(ctx, s.GetRelayerExecReporter(), c.Config().ChainID)
+			s.Require().NoError(err)
 
-		if s.channels[s.T().Name()] == nil {
-			s.channels[s.T().Name()] = make(map[ibc.Chain][]ibc.ChannelOutput)
-		}
+			s.channels[s.T().Name()][c] = channels
```

Comment:
just realised this is wrong! We are currently fetching all channels associated with the chain, not just the one that was created. It means every relayer is still watching all channels, we need to identify the exact channel for this test. I think it is `channels[len(channels)-1]`